cmd/crc/cmd/start.go

```diff
@@ -1,20 +1,23 @@
 package cmd
 
 import (
-	"github.com/spf13/cobra"
+	"fmt"
+	"github.com/code-ready/crc/cmd/crc/cmd/config"
 
+	crcConfig "github.com/code-ready/crc/pkg/crc/config"
 	"github.com/code-ready/crc/pkg/crc/constants"
 	"github.com/code-ready/crc/pkg/crc/errors"
 	"github.com/code-ready/crc/pkg/crc/logging"
 	"github.com/code-ready/crc/pkg/crc/machine"
 	"github.com/code-ready/crc/pkg/crc/preflight"
+	"github.com/spf13/cobra"
 )
 
 func init() {
 	rootCmd.AddCommand(startCmd)
 
 	startCmd.Flags().StringVarP(&bundlePath, "Bundle", "b", constants.DefaultBundle, "The system bundle used for deployment of the OpenShift cluster.")
-	startCmd.Flags().StringVarP(&vmDriver, "VMDriver", "d", constants.DefaultDriver, "The hypervisor driver to use")
+	startCmd.Flags().StringP(config.VMDriver.Name, "d", constants.DefaultVMDriver, fmt.Sprintf("The driver to use for the CRC VM. Possible values: %v", constants.SupportedVMDrivers))
```

Comment:
@praveenkumar Typo here, you missed the variable for the flag.

Reply:
@anjannath didn't get you.

```
$ ./crc start -h
Start cluster

Usage:
  crc start [flags]

Flags:
  -b, --Bundle string      The system bundle used for deployment of the OpenShift cluster. (default "crc_libvirt_v4.1.0-rc0.tar.xz")
  -h, --help               help for start
  -d, --vm-driver string   The driver to use for the CRC VM. Possible values: [virtualbox] (default "hyperkit")

Global Flags:
      --log-level string   log level (e.g. "debug | info | warn | error") (default "info")
```

Reply:
@praveenkumar Sorry my bad, ignore the previous comment, the config values are not binding with the flags so i was getting this error:

```
$ crc start -d libvirt -b ~/Downloads/crc_libvirt_v4.1.0.rc0.tar.xz 
crc - Local OpenShift 4.x cluster
[Unsupported driver: , use '--vm-driver' option to provide a supported driver [libvirt]
]
```
the binding of config values to flags part is missing here: https://github.com/code-ready/crc/blob/1b293c6295f3863261cee591c14f5e2816c359d8/pkg/crc/config/viper_config.go#L49